lib/mix/tasks/install.ex

```diff
@@ -0,0 +1,175 @@
+defmodule Mix.Tasks.Lvn.SwiftUI.Install do
+  @moduledoc "Installer Mix task for LiveView Native: `mix lvn.install`"
+  use Mix.Task
+
+  @requirements ["app.config"]
+
+  @shortdoc "Installs LiveView Native for SwiftUI."
+  def run(args) do
+    {parsed_args, _, _} = OptionParser.parse(args, strict: [host_project_config: :string, task_settings: :string])
+    host_project_config = Keyword.fetch!(parsed_args, :host_project_config)
+    task_settings = Keyword.fetch!(parsed_args, :task_settings)
+
+    make_native_project_dir(host_project_config)
+    copy_xcodegen_files(host_project_config)
+    replace_app_name_in_sources(host_project_config)
+    rename_sources_directory(host_project_config)
+    run_xcodegen(host_project_config, task_settings)
+    remove_xcodegen_files(host_project_config)
+
+    :ok
+  end
+
+  def lvn_install_config, do: %{
+    client_name: "SwiftUI",
+    mix_config: %{
+      live_view_native: [
+        plugins: [LiveViewNative.SwiftUI]
+      ],
+      live_view_native_stylesheet: [
+        parsers: [
+          swiftui: LiveViewNative.SwiftUI.RulesParser
+        ]
+      ]
+    },
+    prompts: [
+      install_xcodegen: %{
+        type: :confirm,
+        label: "Xcodegen is required to generate an Xcode project for your app. Would you like to install it?",
+        ignore: System.find_executable("xcodegen") != nil,
+        on_yes: &install_xcodegen/0,
+        on_no: &skip_swiftui_install/0
+      },
+      targets: %{
+        type: :multiselect,
+        label: "Select any number of deployment targets for your app.",
+        options: ["iOS", "macOS", "watchOS"],
```

Comment:
Can a tvOS option be added here?

Reply:
I don't think we'll be updating core until v0.3. I'm fine with leaving tvOS out from auto-generation until its fully supported.